src/v/cluster/types.h

```diff
@@ -3687,6 +3687,13 @@ struct get_current_cluster_epoch_response
     auto serde_fields() { return std::tie(ec, epoch); }
 };
 
+struct error_info {
+    std::error_code err;
+    ss::sstring message;
```

Comment:
is this different from https://en.cppreference.com/w/cpp/error/error_code/message.html?